fedm05-sp2

cmda9 opened the pull request:
[https://eso.vse.cz/~fedm05/sp02/](https://eso.vse.cz/~fedm05/sp02/)

cmda9 committed:
Add files via upload

cmda9 committed:
commit to the right branch

nvbach91 commented:
mozna to neumim pouzivat, ale nezobrazuje se mi trasa

![image](https://github.com/user-attachments/assets/69c106b5-86da-4179-bb29-7f4c70b41336)

cmda9 commented:
Špecifická trasa se na mapě ani nemá zobrazit, jde spíše o aplikaci pro plánování itineráře tj. ukládání lokalit+datumu příjezdu a odjezdu, samotná trasa která se ukládá se zobrazuje vlevo dolů i s počasím.

nvbach91 commented:
- custom route name
- places api, render,
- async/await fetch() misto xhr,
- geolocation API get coordinates of device, center map
- add new route button
- layout
- nemodifikovat DOM v cyklu
- nevolat tolik pozadavku na weather api, max dates for weather, first and last only

cmda9 commented:
- custom route name
added cursor text indicator to css

- places api, render,
fixed, will get the location of closest city using another api call

- async/await fetch() misto xhr,
done

- geolocation API get coordinates of device, center map - add button to center to current location
Done - reload button

- add new route button - premistnit pro UX
zmena pozice tlacitka, nyni je nad route listem

- layout - levy panel, nech to neni nad tou mapou
vice mista pro routes

- nemodifikovat DOM v cyklu - u route planner
pridan document fragment do ktereho se appenduji routes, nasledne po cyklu updatne DOM timto fragmentem

- nevolat tolik pozadavku na weather api, max dates for weather, first and last only 
max 10 dni, jeslize je vice nez 10 dni, tak se posila pouze zacatek+konec

cmda9 committed:
Add files via upload
